omnibus/files/private-chef-cookbooks/private-chef/libraries/du.rb

```diff
@@ -0,0 +1,25 @@
+require 'mixlib/shellout'
+module Du
+  # Calculate the disk space used by the given path. Requires that
+  # `du` is in our PATH.
+  #
+  # @param path [String] Path to a directory on disk
+  # @return [Integer] KB used by directory on disk
+  #
+  def self.du(path)
```

Comment:
Would it be hard to statfs for this, too? I've got the impression that this would let us stay clear of some potential compatibility issues...